doc/kubernetes/modules/ROOT/pages/running/keycloak-deployment.adoc

```diff
@@ -66,3 +67,5 @@ spec:
           haproxy.router.openshift.io/balance: roundrobin
           haproxy.router.openshift.io/disable_cookies: 'true'
 ----
+
+Take a look at xref:./keycloak-with-external-infinispan-deployment.adoc[Keycloak with external Infinispan deployment] to configure the cache in a remote store.
```

Comment:
likely out of context here

Reply:
Agreed. Will remove it. I was on the fence for this one.